hicommonwealth/commonwealth
# updated privacy page

## Link to Issue
Closes: #8388 

## Description of Changes
- Update Privacy page to reflect new changes

## "How We Fixed It"

-designed the Privacy page off of the google doc provided
 
## Test Plan
- go to `/privacy`
- open this google doc https://docs.google.com/document/d/1ZEwWw2MTBkgxJoRR4GKazbu2jmanrg7g/edit
- confirm that all information is correct and all links open in new tabs
**NOTE: I kept the "Privacy Policy" and "Effective as of 6/8/23" consistent to the `/terms` page to keep some design cohesion.**
